bazel/external_crates.bzl

```diff
@@ -455,6 +455,10 @@ def external_crates_repository(name, cargo_lockfile, lockfile, sanitizers_enable
             "dashmap": crate.spec(
                 version = "^5.3.4",
             ),
+            "der": crate.spec(
```

Comment:
I'm a bit surprised; the IC uses DER left and right do we not already depend on the crate already?